ray-operator/test/support/environment.go

```diff
@@ -0,0 +1,31 @@
+package support
+
+import (
+	"os"
+)
+
+const (
+	// KuberayTestOutputDir is the testing output directory, to write output files into.
+	KuberayTestOutputDir = "KUBERAY_TEST_OUTPUT_DIR"
+
+	// KuberayTestRayVersion is the version of Ray to use for testing.
+	KuberayTestRayVersion = "KUBERAY_TEST_RAY_VERSION"
+
+	// KuberayTestRayImage is the Ray image to use for testing.
+	KuberayTestRayImage = "KUBERAY_TEST_RAY_IMAGE"
+)
+
+func GetRayVersion() string {
+	return lookupEnvOrDefault(KuberayTestRayVersion, RayVersion)
+}
+
+func GetRayImage() string {
+	return lookupEnvOrDefault(KuberayTestRayImage, RayImage)
```

Comment:
I am not sure about how you feel about adding logic to support arm64 images. IMHO, using the `-aarch64` should be the case if we detect that the host the test is run on has that architecture. I have simillar code in the apiserver e2e test (PR)

Reply:
While it would solve the problem locally, I think this should be addressed holistically by publishing multi-architecture container images, with the tags pointing to the multi-architecture manifests, so the right image is resolved by the container runtime when pulling the image. I've created #1630 to track that.